swagger.yaml

```
@@ -6937,7 +6963,7 @@ definitions:
         description: Name of the version to be presented to the user.
       support_level:
         type: string
-        enum: [beta, production, maintenance]
+        enum: [beta, production, maintenance, end of life]
```

```suggestion
        enum: [beta, production, maintenance, end-of-life]
```

to match the convention we're using for support-level